dnf/cli/commands/__init__.py

```diff
@@ -226,7 +226,7 @@ class ProvidesCommand(Command):
     provides command.
     """
 
-    aliases = ('provides', 'whatprovides')
+    aliases = ('provides', 'whatprovides', 'prov', 'wprov',)
```

Comment:
Please only ``prov`` because alias ``whatprovides`` was not previously described